src/ontology/cl-edit.owl

```diff
@@ -10043,7 +10053,7 @@ AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000767 "CALOHA:TS-0073")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000767 "FMA:62862")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000767 "basophilic leucocyte")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000767 "basophilic leukocyte")
-AnnotationAssertion(rdfs:comment obo:CL_0000767 "Matures in the bone marrow and account for <1% of leukocytes in the peripheral blood, spleen, and bone marrow. Basophils are described as being CD11a-positive, CD11b-positive, CD13-positive, CD15-positive, CD18-positive, CD21-positive, CD25-positive, CD29-positive, CD35-positive, CD40-positive, CD40L-positive, CD44-positive, CD45R-negative, CD46-positive, CD49a-positive, CD49b-positive, CD49d-positive, CD55-positive, CD59-positive, CD62L-positive, CD63-positive, CD69-positive, CD90-negative, CD116-positive, CD117-negative, CD124-positive, CD125-positive, CD131-positive, CD161-positive, CD184-positive, CD191-positive, CD192-positive, CD197-positive, CD200R3-positive, CD218-positive, CD282-positive, CD284-positive, CD289-positive, CD290-positive, CD294-positive, natural killer cell receptor 2B4-positive, smad1-positive, CD3-negative, CD4-negative, CD7-negative, CD8-negative, CD14-negative, CD15-negative, CD16-negative, CD19-negative, CD20-negative, CD34-negative, CD36-negative, CD45R-negative, CD56-negative, CD57-negative, CD235a-negative, and GR1-negative. Transcription factors- GATA1-positive, PU.1-positive.")
```

Comment:
I think this change was not intentioned (it removed a '<1%')

Reply:
It would be good to write down this instead of using symbols. :)

Reply:
@aleixpuigb where the '<1%' should have been in that line? before "Mature in the bone marrow... ? 
so I can add it again

Reply:
"Matures in the bone marrow and account for <1% of leukocytes" tou can substitute for "less than 1 per cent".

Reply:
Thank you I will fix this